index.html

```diff
@@ -29,84 +29,68 @@
-            <p class="module-links"><a href="https://github.com/AmpersandJS/ampersand-select-view">github</a><a href="https://www.npmjs.org/package/ampersand-select-view">npm</a></p>
-          </div><a href="/contribute" class="button button-primary">How to contribute</a>
-        </div>
+        <h2>What is Ampersand.js?</h2>
+        <p>Imagine a version of <a href="http://backbonejs.org/">Backbone.js</a> where each piece (model, view, collection, etc) is its own <a href="http://github.com/ampersandjs">GitHub repository</a> and <a href="https://www.npmjs.com/search?q=ampersand">npm module</a> with good, unified <a href="/docs">documentation</a> (and yes, npm is <em>wonderful</em> for frontend code).</p>
+        <p><em>But</em>, it's not just that. We added features to some of those pieces and <a href="http://ampersandjs.com/docs#ampersand-state">replaced others entirely</a> but if you know Backbone.js you'll feel at home.</p>
+        <p>Companies like <a href="https://www.whatsapp.com/">WhatsApp</a>, <a href="http://www.flipkart.com/">FlipKart</a>, and <a href="#who">others</a> have shipped apps at incredible scale using various Ampersand modules.</p>
+        <p>There is no "core" module. You <code>npm install</code> only what you need.</p>
+        <p>We didn't set out to make a framework. <a href="http://andyet.com/">&yet</a> has been using this toolset to build apps for clients and ourselves for years (i.e. <a href="http://talky.io/">Talky</a>). Eventually we documented <a href="https://blog.andyet.com/2014/06/25/introducing-ampersand-js">and released</a> it with an MIT license.</p>
+        <p>Ampersand.js was heavily influenced by the Node.js community and <a href="https://blog.andyet.com/2015/04/14/frontend-frameworks-to-pair-with-node">pairs well</a> with teams that already use Node.js.</p>
+        <p>We support modern browsers (for IE that means IE9+).</p>
+        <h3>Maximize simplicity</h3>
+        <p><em>If you don’t actively fight for simplicity in software, complexity will win, and it will suck.</em></p>
+        <h2>Key items to understand</h2>
+        <ul>
+          <li>Ampersand.js is designed to be used with a module bundler like <a href="http://browserify.org/">Browserify</a> or <a href="http://webpack.github.io/">Webpack</a>. If you've never used one of those read our <a href="http://ampersandjs.com/learn/npm-browserify-and-modules">intro to npm, Browserify, and Modules</a>.</li>
+          <li>Ampersand.js is optimized for mixing-and-matching with other stuff. Financial Times used ampersand-state an ampersand-view combined with <a href="http://d3js.org/">D3.js</a> to make a <a href="http://www.ft.com/ig/sites/2014/debt-to-gdp-ratio/#usa">world debt visualizer</a>. Both <a href="https://www.whatsapp.com/">WhatsApp</a> and <a href="http://www.flipkart.com/">FlipKart</a> use React as the view layer. Having that level of flexibility is one of the main reasons we created Ampersand.js to begin with.</li>
```

Comment:
@HenrikJoreteg: "Financial Times used ampersand-state an ampersand-view" -> "Financial Times used ampersand-state and ampersand-view"